timeline/views_ajax.py

```diff
@@ -65,6 +169,27 @@ def get(self, request, *args, **kwargs):
         return Response(self.get_event_details(event), status=200)
 
 
+class ProjectEventExtraAjaxView(EventExtraMixin, SODARBaseProjectAjaxView):
+    """Ajax view for retrieving event extra data for projects"""
+
+    permission_required = 'timeline.view_timeline'
+    allow_anonymous = True
```

Comment:
`allow_anonymous` should not be set true in these views. This is not your fault as it seems to be incorrect in the existing event detail ajax views as well. Feel free to remove.

Also, please add permission tests for these new ajax views under `timeline.tests.test_permissions_ajax`.

Reply:
Deleteted this line